x-pack/plugins/maps/public/kibana_services.d.ts

```diff
@@ -12,7 +12,6 @@ export function getFileUploadComponent(): any;
 export function getIndexPatternSelectComponent(): any;
```

Comment:
Now that `MapsConfigType` is defined, use `MapsConfigType` types in setMapConfig and getMapConfig